### ISSUE TITLE
Search option on traccar 5.3 is not available or doesn´t work

### ISSUE THREAD

**Issue by cristianp431** [traccar/traccar-web#1021]
**Is your feature request related to a problem? 

If I have a hundreds of devices an users sometime is difficult to find them because in each field we can´t type info to search that is happening on web app new interface in old interface its fine

**Describe the solution you'd like**

its possible to add a search option for users, and devices like on old web app version? In the new version there is not a check list to add devices to a particular user or search users or devices.

**Describe alternatives you've considered**

Create a chek list box like old web version

**Comment by tananaev** [comment]
I don't think we can add a check list like in the old version, but we can implement a search.

**Comment by tananaev** [comment]
Recent versions support search in linking.
